xAPI.md

```diff
@@ -509,11 +509,47 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 8.0 Getting More Out of xAPI
 
+xAPI focuses on the structure of communication between a client and an LRS.  It spares many details of the content to
+increase flexibility.  The intention in only supplying a basic amount of syntax rules is that Communities of Practice (CoP) can define content (verbs, activity types, contextual relationships, extensions, etc.) for their use cases.  It is very 
+important that such communities exist and share best practices.
+
```

Comment:
Not sure about the phrasing of "basic amount of syntax rules" but otherwise +1 to this paragraph.